test/cqlpy/test_bloom_filter.py

```diff
@@ -3,9 +3,8 @@
 # SPDX-License-Identifier: AGPL-3.0-or-later
 
 import pytest
-import rest_api
-import nodetool
-from util import new_test_table, config_value_context
+from . import nodetool, rest_api
```

Comment:
nitpick: I would prefer having two separate import lines for nodetool and rest_api. Those are two separate files - not two functions in one file as this syntax seems to suggest.